### PR TITLE
Extend text element

### PR DESCRIPTION
**Pull request by aju**
closes https://github.com/brainly/style-guide/issues/1341

### CODE REVIEW

File: src/components/text/Text.jsx [hunk 1]
```diff
@@ -40,13 +40,41 @@ const WEIGHT = {
   BOLD: 'emphasised'
 };
 
-const Text = ({children, size, weight, color, className, type = TYPE.DIV, noWrap, breakWords, ...props}) => {
+const TEXT_TRANSFORM = {
```
Comment (burakukula) [inline src/components/text/Text.jsx]: @aju TEXT_WEIGHT, TEXT_COLOR, TEXT_SIZE?

Comment (burakukula) [reply src/components/text/Text.jsx]: @aju or it would be too big refactor for now?

**Comment by aju** [comment]
@vergilius 
screen shot 2018-01-05 at 15 08 23
screen shot 2018-01-05 at 15 08 39

File: src/components/text/Text.jsx (hunk 1 above)
Comment (aju) [reply src/components/text/Text.jsx]: One step at a time:P

**Pull request merged**